### PR TITLE
3.next - Scoped middleware docs

### PR DESCRIPTION
**Pull request by markstory**
I've not documented the Application class hook yet, as I want to think more about how to document that class more holistically.

Refs: cakephp/cakephp#10646

### CODE REVIEW

File: en/development/routing.rst [hunk 1]
```diff
@@ -753,6 +754,39 @@ Then to create links which map back to the routes simply use::
...
+----------------------------
+
+Middleware can be applied to your entire application, or to an individual
+routing scope. Before middleware can be applied to a scope, it needs to be
+registered::
+
+    use Cake\Http\Middleware\CsrfProtectionMiddleware;
+    use Cake\Http\Middleware\EncryptedCookieMiddleware;
+
+    Router::scope('/', function ($routes) {
```
Comment (saeideng) [inline en/development/routing.rst]: `//in config/routes.php` ?

Comment (markstory) [inline en/development/routing.rst]: Done!

**Comment by markstory** [comment]
Not sure I understand what you mean @ravage84

**Comment by ravage84** [comment]
I just meant @saeideng's remark

> //in config/routes.php ?

So, we're fine, @markstory.

**Pull request merged**